frontends/krita/krita_comfy/prompt.py

```diff
@@ -0,0 +1,327 @@
+from dataclasses import dataclass, field
+from enum import Enum, unique
+from typing import List
+import itertools
+import json
+import re
+import copy
+
+from krita import (
+    QImage,
+    QByteArray,
+    QSize,
+    QBuffer,
+    QIODevice
+)
+
+from .config import DEFAULTS
+from .defaults import (
+    DEFAULT_NODE_IDS
+    , PROMPT
+    , PRUNED_DATA
+)
+
+def _default(self, obj):
+    return getattr(obj.__class__, "to_json", _default.default)(obj)
+
+_default.default = json.JSONEncoder().default
+json.JSONEncoder.default = _default
+
+
+def get_members(obj):
+    return [attr for attr in dir(obj) if not callable(getattr(obj, attr)) and not attr.startswith("__")]
+
+
+def bind_or_none(data: dict, keys: list[str]):
```

Comment:
```suggestion
def bind_or_none(data: dict, keys: List[str]):
```
Getting an error of type not being suscriptable.

I suggest using the classes from the `typing` module.  You'll need to import `Tuple` as well.
Lines 35, 216, 220 are causing this error.

Reply:
Thank you I see
# On Python 3.8 and earlier, the name of the collection type is
# capitalized, and the type is imported from the 'typing' module

Reply:
As for the tuple looks like variable size is only on 3.9 so I'll stick with a list.
`# For tuples of variable size, we use one type and ellipsis`
`x: tuple[int, ...] = (1, 2, 3)  # Python 3.9+`